crates/dtrace_probe/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "dtrace_probe"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+color-eyre = "0.6"
+composer = { path = "../composer" }
```

Since #21 has been merged this has to be updated

```suggestion
composer_api = { path = "../composer_api" }
```

Sorry for the hassle. .)